test-requirements.txt

```diff
@@ -1,3 +1,4 @@
 pytest==8.3.3
 pytest-asyncio==0.24.0
 pytest-mock==3.14.0
+ruamel.yaml
```

Comment:
Nit: Do we not want to pin this to a specific version?

Reply:
I have replaced it with pyyaml which is already present for OpenFL. Removed this entry.